assets/js/ledyer-checkout-for-woocommerce.js

```diff
@@ -443,24 +435,15 @@ jQuery(function ($) {
 
             $(document).on('ledyerCheckoutOrderComplete', function (event) {
                 lco_wc.logToFile('ledyerCheckoutOrderComplete from Ledyer triggered');
-
-                if (lco_wc.redirectUrl !== null) {
```

Comment:
This was put in place in order to avoid duplicate orders, make sure to verify this in HK staging